ui/package.json

```diff
@@ -10,6 +10,10 @@
     "lint:fix": "turbo run lint -- --fix",
     "type-check": "turbo run type-check",
     "start": "turbo run start",
+    "start-with-storybook": "turbo run start storybook:start",
+    "storybook": "turbo run storybook:start",
+    "storybook:build": "turbo run storybook:build",
+    "storybook:serve": "turbo run storybook:serve",
```

Comment:
Trying to think about some of this. Do folks want to always start both the storybook and the app or will they mostly want to start them separately?

Reply:
Separately works now

Reply:
Separately is how I'd think about it... I'd run storybook when I need it (because I'm developing locally and updating stories)

The other way I use Storybook is for reference, and in that usage, I like the idea of having a permanent URL to login to and see the current state of Storybook

Reply:
> The other way I use Storybook is for reference, and in that usage, I like the idea of having a permanent URL to login to and see the current state of Storybook

When we get to a place where we're happy with the results, I'm hoping we'll build and publish a static version of the storybook to github pages on `merge-to-master` for this purpose.